### DISCUSSION TITLE
Can't select default chain

### DISCUSSION THREAD

**Discussion by romainjacquel** [arcana-network/discussions#20] [Arcana Dashboard]
I've fill the app creation form in arcana dashboard but i can't select an "Default Chain". And i can't create my app :/

Video example: https://www.loom.com/share/8bf863ff4d134a3caf8b81a99d6a7de5?sid=2fd64025-aaab-49c0-8ab9-36be37139ea9

**Comment by anivenu** [comment]
@romainjacquel Can you please try again now and let us know if it persists? 
If you do hit it again can you show us the error in the console?

**Reply by romainjacquel** [reply]
Hi, thanks for your response. I have the same issue, my browser cache is cleaned and my browser is up-to-date. I'm on windows 11 and there is google chrome version.

<img width="536" alt="image" src="https://github.com/arcana-network/discussions/assets/43607419/da84f6c6-aff5-4118-becd-8a21cd0eb16f">

And a snapshot of my console:

<img width="1094" alt="image" src="https://github.com/arcana-network/discussions/assets/43607419/e05237f5-ec05-4ecf-bdaa-4204d3326084">
